### PR TITLE
Update readme with composer info

### PR DESCRIPTION
**Pull request by richeduni**

### CODE REVIEW

**Comment by Sephster** [comment]
This pull request adds information to the installation instructions about installing dependency packages via composer. This is a required step

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
```
Comment (Sephster) [inline CHANGELOG.md:8]: Shouldn't need to add this to the changelog as it is meta data :+1:

File: README.md [hunk 2]
```diff
@@ -41,13 +41,26 @@ If the Multisite Privacy plugin or the More Privacy Option plugins are installed
 
 1. Copy the ed-lti folder to the plugins folder.
 **Note:** for instructions on how to install a plugin manually, please visit [WordPress for dummies](https://www.dummies.com/web-design-development/wordpress/templates-themes-plugins/how-to-install-wordpress-plugins-manually/)
-2. From **Network Admin > Plugins** select the plugin and click **Network Activate**.
-3. Go to **Network Admin > Settings > LTI Platforms** and enter a Name, Issuer (URL of connecting VLE, e.g. http://moodle.test) Client ID, Deployment ID, Authentication Request URL, Access Token URL, Public Keyset URL. These properties should be provided by the VLE. For example, on moodle, once you have created a tool, click on the 'View configuration details' icon to work out what values to enter for the platform.
+2. Either locally (before you push the plugin) or on the server, you will need to run composer to install the imsglobal libraries.
```
Comment (Sephster) [inline README.md:44]: can we change this to provide a bit more information for people that might not be familiar with Composer? Something like:

> ...you will need to run [Composer](https://getcomposer.org), a package manager for PHP, to install all dependency libraries for this plugin.

**Review by Sephster — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (richeduni) [reply CHANGELOG.md]: Reverted back

File: README.md (hunk 2 above)
Comment (richeduni) [reply README.md]: Added - or something similar

File: CHANGELOG.md (hunk 1 above)
Comment (Sephster) [reply CHANGELOG.md]: Thanks Richard.

File: README.md (hunk 2 above)
Comment (Sephster) [reply README.md]: Cheers!

**Comment by Sephster** [comment]
Fixes issue #4

**Pull request merged**